src/monitors/client_metrics.rs

```diff
@@ -57,6 +59,9 @@ impl From<ObTableOperationType> for ObClientOpRecordType {
             ObTableOperationType::Replace => ObClientOpRecordType::Replace,
             ObTableOperationType::Increment => ObClientOpRecordType::Increment,
             ObTableOperationType::Append => ObClientOpRecordType::Append,
+            ObTableOperationType::CheckAndInsertUp => ObClientOpRecordType::CheckAndDo,
+            ObTableOperationType::Invalid => ObClientOpRecordType::Invalid,
+            _ => unimplemented!(),
```

Comment:
Avoid the default pattern, it may lead to some implicit errors when a new type is added.

Reply:
Fixed, all type has corresponding ObClientOpRecordType. Some types that haven't been used have been set to unreachable() now.